ddtrace/tracer/textmap.go

```diff
@@ -380,6 +380,19 @@ func unmarshalPropagatingTags(ctx *spanContext, v string) {
 	}
 }
 
+// unmarshalPropagatingTags unmarshals tags from v into ctx
+func appendPropagatingTags(ctx *spanContext, k, v string) {
+	if ctx.trace == nil {
```

Comment:
Is this a use case we need to cover (e.g. trying to set propagating tags before a trace exists)?

Reply:
When extracting, we are creating a new trace context : `var ctx spanContext` within the Extract function. So, in that case, it will be `nil`